### PR TITLE
Feature/overriding refresh token timeouts

### PR DESCRIPTION
**Pull request by rp4fx12**
There might be a better approach here (or this may not be desired or needed at all) but this should allow for users to override the advisory and mandatory refresh timeouts that are a part of the RefreshableToken Class - addresses this issue: #2189

### CODE REVIEW

File: botocore/credentials.py [hunk 1]
```diff
@@ -406,6 +400,22 @@ def create_from_metadata(cls, metadata, refresh_using, method):
         )
         return instance
 
+    @property
+    def _advisory_refresh_timeout(self):
```
Comment (rp4fx12) [inline botocore/credentials.py:404]: Not sure if this is an acceptable way to handle this but just offering it as a potential suggestion.

Comment (rp4fx12) [reply botocore/credentials.py:404]: Might not also want to pull from the environment variables every time this property is called.

Comment (rittneje) [reply botocore/credentials.py:404]: Why not just save it off in the constructor rather than making a property method?

**Comment by aBurmeseDev** [comment]
Thank you for submitting this PR. After team discussion, we're going to have to discuss with other SDKs for potential cross-SDK approach to keep it consistent and decided not to consider this one at this time.

**Pull request closed without merging**